src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/features/form/containers/GroupContainer.tsx

```diff
@@ -88,6 +88,10 @@ export function GroupContainer({
     if (container.edit?.multiPage) {
       setMultiPageIndex(0);
     }
+
+    if (container.openDefault == true && repeatingGroupIndex == -1) {
```

Comment:
I prefer explicit checks here `===` 
```suggestion
    if (container.openDefault === true && repeatingGroupIndex === -1) {
```